### DISCUSSION TITLE
CF drifting forward/left when hovering

### DISCUSSION THREAD

**Discussion by Itai51** [bitcraze/discussions#1441] [Q&A]
Hello CF forum,

I finally got my [setup](https://github.com/orgs/bitcraze/discussions/1391#discussioncomment-9858187) working after recalibrating it to work with 3 out of 4 lighthouses in my lab! In fact, it worked so well I was able to fly in a few different trajectories and I was really satisfied with the performance.

Recently, I've been tasked with taking stateEstimate, acc, and gyro measurements while hovering in-place. After powering on the fourth lighthouse and recalibrating, I now find that during the 20-30 seconds that my CF is supposed to hover in-place, it actually drifts 0.5-1.0 meters in both the X and Y directions (forward and to the left) . Since I determined in the linked comment thread that my setup works, and since the CF was flying fine until this last task, I am struggling with troubleshooting this issue.

In short, why is my CF drifting forward/left, and how do I fix it?

Thank you!

**Comment by knmcguire** [comment]
Hmm so as you say, once you turn on the 4th basestation, the Crazyflie suddenly flies to a different location? Or does it actually drift away? What happens with the LEDS on top of the lighthouse deck? Does it stay green or does it turn orange when it drifts?

Also, have you checked the estimated position of the 4th basestation? Does it align with it's actual position in space?
